source/release-notes/1.7.txt

```diff
@@ -15,6 +15,62 @@ Release Notes for mongosync 1.7
 Patch Releases
 --------------
 
+.. _1.7.3-c2c-release-notes:
+
+1.7.3 Release
+~~~~~~~~~~~~~
+
+**June 5, 2024**
+
+Issues Fixed:
+
+- Fixed bug introduced in v1.5.0 where mongosync could miss change
+  events when reading documents from a lagging secondary node and the
+  Oplog Rollover Resilience mechanism is enabled.
+
+- Fixed bug introduced in v1.0.0 where the destination may have missing
+  or extra indexes after mongosync is committed. This is an unlikely
+  edge case.
+
+- Fixed bug introduced in v1.0.0 where mongosync may miss change events
+  for a document. This is an unlikely edge case that is only possible if
+  the destination server version is MongoDB 5.0 or earlier.
+
+- Fixed bug introduced in v1.0.0 that could result in size mismatches
+  between source and destination capped collections. This can only
+  affect migrations if all of the following conditions are met:
+  
+  - The source cluster is MongoDB 6.0 or earlier.
+  
+  - The destination cluster is MongoDB 7.0 or later.
+  
+  - A capped collection's size was modified during a migration.
+
+- Fixed bug introduced in v1.0.0 where mongosync ignored the
+  ``changeStreamPreAndPostImages`` field in ``modify`` change events.
+  This could result in a destination collection having a different value
+  for this field compared to its source collection. mongosync now
```

Comment:
```suggestion
  for this field compared to its source collection. Mongosync now
```
We're not super consistent with this on our end, but by itself, we tend to treat the word "mongosync" like a common noun unless we're explicitly code-stylizing it like `mongosync`.

I see that most other mentions in the public docs stylize it, so I'd be in favor of doing a broader update where we change mongosync/Mongosync to `mongosync`  for consistency's sake. It's probably better suited for a separate ticket if you agree -- I'm okay with whatever you decide here in the meantime.

Reply:
Makes sense to me, thanks for pointing this out. I'll capitalize here per your suggestion and can create a follow-up ticket to standardize across the docs.

Reply:
Here is the ticket to handle the standardization: https://jira.mongodb.org/browse/DOCSP-40158